RELEASING.md

```diff
@@ -0,0 +1,25 @@
+# Releasing
+
+Follow these steps to release a new version of bootstrap_form to rubygems.org.
+
+## Prerequisites
+
+* You must have commit rights to the bootstrap_form repository.
+* You must have push rights for the bootstrap_form gem on rubygems.org.
+* You must be using Ruby >= 2.2.
+* Your `~/.netrc` must be configured with your GitHub credentials, [as explained here](https://github.com/mattbrictson/chandler#option-1---netrc).
```

Comment:
Does this *have* to be option 1 of the Github credentials, or can I use one of the other options documented for chandler?

Reply:
No, any of Chandler's authentication methods should work. I'll clarify the doc.